projects/DETR3D/configs/detr3d_r101_gridmask.py

```diff
@@ -3,7 +3,7 @@
     'mmdet3d::_base_/default_runtime.py'
 ]
 
-custom_imports = dict(imports=['projects.detr3d.detr3d'])
+custom_imports = dict(imports=['projects.DETR3D.detr3d'])
```

Comment:
Line 241: load_from='pretrained/fcos3d.pth' 
since you describe the saved path is `pretrained/` in README.md.
Better to add `--cfg-options load_from=pretrained/fcos3d.pth` in the training command.